Dockerfile

```
@@ -1,4 +1,4 @@
-FROM rocker/binder:4.3.2
+FROM rocker/binder:4.3.3
```

```suggestion
FROM rocker/binder:4.3.2
```

This suggestion can be merged to make us get a rebuild on merge without any actual change to anything in the repo.